src/utils/formatNumbers.ts

```diff
@@ -33,3 +33,24 @@ export const currencyFormatter = (
     return formatter.format(value);
   };
 };
+
+export function numberWithCommas(num?: string | number) {
+  if (num === undefined || num === null) {
+    return;
+  }
+
+  if (typeof num === 'string') {
+    num = parseFloat(num);
+  }
+
+  // Check if the number has decimals
+  const hasDecimal = num % 1 !== 0;
+
+  return num.toLocaleString('en-US', {
```

Comment:
we do have a decimalExt formatter already

Reply:
done

Reply:
done